source/state_representation/README.md

```diff
@@ -159,6 +159,53 @@ StateRepresentation::CartesianTwist wVa("a", Eigen::Vector3d(1, 0, 0));
 StateRepresentation::CartesianPose wPa = period * wVa; // note that wVa * period is also implemented
 ```
 
+### Cartesian state distance and norms
+
+As a `CartesianState` represents a spatial transformation, distance between states and norms computations have been
+implemented. The distance functions is represented as the sum of the distance over all the state variables:
+
+```cpp
+using namespace StateRepresentation;
+CartesianState cs1 = CartesianState::Random("test");
+CartesianState cs2 = CartesianState::Random("test");
+
+double d = cs1.dist(cs2);
+// alternatively one can use the friend type notation
+d = dist(cs1, cs2)
+```
+
+By default, the distance is computed over all the state variables.
```

Comment:
I think it's worth stating that the "distance" unit is an abstract quantity when used on a whole state object, and giving an example of why it's a useful quantity to have / compare.